### PR TITLE
Changes for RAGL S11.

### PR DESCRIPTION
**Pull request by tttppp**
Currently this is missing the updated rules for RAGL S11 and the scheduling is different this season.

### CODE REVIEW

File: laddertools/ragl-s11.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+        - [9877,  SinJul]
+        - [6430,  Blackened]
+        - [11435, Goremented]
+        - [7644,  KSK_Nico]
+        - [3952,  anjew]
+        - [7304,  Upps]
+        - [4527,  FiveAces]
+        - [8869,  Sigil]
+
+    Minions Alfa:
```
Comment (ubitux) [inline laddertools/ragl-s11.yml:21]: Alfa, not Alpha?

File: raglweb/templates/info.html [hunk 2]
```diff
@@ -26,10 +26,11 @@ <h2>The essentials</h2>
 <dl>
 	<dt>Game version:</dt><dd><a href="https://github.com/OpenRA/OpenRA/releases/tag/release-20210321">release-20210321</a></dd>
 	<dt>Game server instances:</dt><dd>|ragl.org| RAGL Server <i>N</i></dd>
-	<dt>Map pack:</dt><dd><a href="{{ url_for('static', filename='ragl-map-pack-2020-12-28.zip') }}">2020-12-28</a></dd>
+	<dt>Map pack:</dt><dd><a href="{{ url_for('static', filename='ragl-map-pack-2020-10-09.zip') }}">2020-10-09</a></dd>
```
Comment (ubitux) [inline raglweb/templates/info.html]: 2021

Also, there is another link to that map pack at the bottom (we probably want that in the configuration at some point to reduce the risk of forgetting one).

File: laddertools/ragl-s11.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+        - [12261, DVoid]
+        - [11913, Duke Bones]
+        - [7591,  manta151]
+        - [6751,  Tailix Killa Mentor]
+        - [12654, Therapist]
+        - [6861,  potato]
+
+Forfeit: []
+
+Playoffs: {}
```
Comment (ubitux) [inline laddertools/ragl-s11.yml:46]: Are these entries mandatory?

File: laddertools/ragl-s11.yml (hunk 1 above)
Comment (tttppp) [reply laddertools/ragl-s11.yml:21]: Ah yep. Last season we had a mix of "alpha, beta" and "alpha, bravo". Previous seasons seemed to use "alfa, bravo, charlie, delta" so I stuck with that.

File: laddertools/ragl-s11.yml (hunk 3 above)
Comment (tttppp) [reply laddertools/ragl-s11.yml:46]: No - I'll remove them.

**Review by ubitux — CHANGES_REQUESTED** [review]
Alright we're almost good. Can you:

1. Add `ragl-s11.yml` to the `MANIFEST.in` (so that it's packaged in the wheel when deploying
2. Squash all your commits together

I'll merge after that.

**Pull request merged**